contracts/internal/balances/ExternalLending.sol

```diff
@@ -106,6 +105,14 @@ library ExternalLending {
                     forRedemption - oracleData.externalUnderlyingAvailableForWithdraw
                 );
             }
+        } else if (targetAmount > oracleData.currentExternalUnderlyingLend) { 
+            // in case of deposit , check maxExternalDeposit (limit due to the supply cap on external pools)
+            uint256 forDeposit = targetAmount - oracleData.currentExternalUnderlyingLend;
+            if (forDeposit > oracleData.maxExternalDeposit) {
+                targetAmount = targetAmount.sub(
```

Comment:
Can you flip the inequalities for consistency?

`oracleData.currentExternalUnderlyingLend < targetAmount`

```
targetAmount = targetAmount - (targetAmount - currentExternalUnderlyingLend - maxExternalDeposit)
targetAmount = maxExternalDeposit - currentExternalUnderlyingLend
```